packages/venia-concept/src/components/Navigation/navigation.js

```diff
@@ -11,6 +11,7 @@ import defaultClasses from './navigation.css';
 import NavHeader from './navHeader';
 import Button from 'src/components/Button';
 import Icon from 'src/components/Icon';
+import { createAccount } from 'src/actions/user';
 
 const CATEGORIES = [
```

Comment:
I'm sorry if I forgot, but did we discuss using a GraphQL query here to retrieve top-level categories?

Reply:
I guess no, it was not the part of tasks we are working on.